docs/INSTALLATION.md

```diff
@@ -1,2 +1,13 @@
 # Installation
 
+Install the essential build tools, the QEMU emulator, and the GCC ARM64 cross-toolchain to test your code.
+
+```shell
+$ sudo apt install build-essential qemu-user qemu-user-static binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu
+```
+
+The makefile should be able to handle the rest.
+
+## Special note for Windows users
+
+Use Windows Subsystem for Linux's Ubuntu image to do the same steps listed above.
```

Comment:
Does it matter if it's WSL or WSL2?

Reply:
WSL2 would probably be better for this now that I think of it.